src/utilities/fetch-package-repos.js

```diff
@@ -18,7 +18,6 @@ const fetch = {
       hides: excludedLoaders,
     },
     'babel/babel-loader',
-    'peerigon/extract-loader',
     'Banno/polymer-webpack-loader',
```

Comment:
I think removing `polymer-webpack-loader` will be good too, it is unused many developers and don't updated